views/add_event.hbs

```diff
@@ -85,27 +57,21 @@
       <option value="wedding">Wedding</option>
     </select>
 
-    <label>Accessibility: </label>
-    <select multiple class="accessibility form-control" name="accessibilityOptions" required>
-      <option disabled selected>Select Accessibility..</option>
-      <option value="Audio recordings">Audio Recordings</option>
-      <option value="Big fonts">Big Fonts</option>
-      <option value="Braille">Braille</option>
-      <option value="Carer">Carer</option>
-      <option value="Disabled parking">Disabled Parking</option>
-      <option value="Disabled toilets">Disabled Toilets</option>
-      <option value="Good lighting">Good Lighting</option>
-      <option value="Place for guide dog">Place for Guide Dog</option>
-      <option value="Sign language">Sign Language</option>
-      <option value="SMS messaging">SMS Messaging</option>
-      <option value="Wheelchair access">Wheelchair Access</option>
-    </select>
+    <label>Place </label>
+    <li id="addPlaceIcon" class="glyphicon glyphicon-plus"></li>
```

Comment:
Why is this a list element?

Reply:
no idea ill change it to a div doesn't make a difference lol